packages/ra-data-graphql-simple/src/buildVariables.test.ts

```diff
@@ -173,3 +173,181 @@ describe('buildVariables', () => {
         });
     });
 });
+
+describe('buildVariables with meta param', () => {
+    const introspectionResult = {
+        types: [
+            {
+                name: 'PostFilter',
+                inputFields: [{ name: 'tags_some' }],
+            },
+        ],
+    };
+    describe('GET_LIST', () => {
+        it('returns correct variables', () => {
+            const params = {
+                filter: {
+                    ids: ['foo1', 'foo2'],
+                    tags: { id: ['tag1', 'tag2'] },
+                    'author.id': 'author1',
+                    views: 100,
+                },
+                pagination: { page: 10, perPage: 10 },
+                sort: { field: 'sortField', order: 'DESC' },
+                meta: { sparseFields: [] },
+            };
+
+            expect(
+                buildVariables(introspectionResult)(
+                    { type: { name: 'Post', fields: [] } },
+                    GET_LIST,
+                    params,
+                    {}
+                )
+            ).toEqual({
+                filter: {
+                    ids: ['foo1', 'foo2'],
+                    tags_some: { id_in: ['tag1', 'tag2'] },
+                    author: { id: 'author1' },
+                    views: 100,
+                },
+                page: 9,
+                perPage: 10,
+                sortField: 'sortField',
+                sortOrder: 'DESC',
+                meta: { sparseFields: [] },
+            });
+        });
```

Comment:
According to the previous comment, the assertion should look like: 
```suggestion
            ).toEqual({
                filter: {},
                meta: { sparseFields: [] },
            });
        });
```
Same for the other tests